sklearn/metrics/tests/test_score_objects.py

```diff
@@ -64,7 +64,7 @@
 
 def _make_estimators(X_train, y_train, y_ml_train):
     # Make estimators that make sense to test various scoring methods
-    sensible_regr = DummyRegressor(strategy='median')
+    sensible_regr = DecisionTreeRegressor(random_state=0)
```

Comment:
@jnothman Thanks. After modification, the test takes around 0.1s on my PC, only slightly longer than before. In Travis, the test is not among the most time consuming tests. For the default parameter of random forest, n_estimators(10) is small, considering the number of training samples(100) is also small, the execution time seems to be acceptable.